test/cppsuite/test_harness/workload_generator.h

```diff
@@ -106,6 +108,10 @@ class workload_generator : public component {
             testutil_check(
               session->open_cursor(session, collection_name.c_str(), NULL, NULL, &cursor));
             for (size_t j = 0; j < key_count; ++j) {
+                /* Generated keys are based on the index for now. */
+                generated_key = std::to_string(j + 1);
```

Comment:
Is the plan to change this when we work on the update ticket?

Reply:
Keys are now incremented, starting from "0". Zero's are appended to the front if the current key value is less than the requested key size.